docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: '3.8'
+
+services:
+  adrenaline:
+    build: .
+    ports:
+      - '3000:3000'
+    volumes:
+      - .:/app
+      - /app/node_modules
+    environment:
+      - CHOKIDAR_USEPOLLING=true
```

Comment:
Since this compose file only has a single service, it'd probably be worth removing and just listing the neccesary `docker run` command in the README:
```
docker build --tag adrenaline . --load && \
	docker run --rm \
	-p 3000:3000 \
	--volume .:/app \
	--volume /app/node_modules \
	--env CHOKIDAR_USEPOLLING=true \
	adrenaline
```

Reply:
However, using docker-compose.yml makes it easy to configure and extend. I find it interesting to have both, and you @Nezteb ?

Reply:
This was my rationale

Reply:
> However, using docker-compose.yml makes it easy to configure and extend. I find it interesting to have both, and you @Nezteb ?

This was my rationale. Usually why I prefer docker-compose even for simple things.

Reply:
I only suggested `docker build` because my host of choice is [Fly.io](https://fly.io/) and they don't support deploying via `docker compose`; you can only use a single `Dockerfile` (with some extra config on top).

That being said:
![both](https://user-images.githubusercontent.com/3588798/229254556-a0ab3c79-84fd-438d-80b3-0099b435a568.gif)